### DISCUSSION TITLE
Is there a way to change mappings based on current ide (Rider vs Intellij)

### DISCUSSION THREAD

**Discussion by TristanLeclair** [JetBrains/ideavim#601] [Q&A] [answered]
I develop in both Java and C# and I recently got into Rider though I've been using Intellij for a while. They both source `.ideavimrc` from the same place but they don't support the same actions. Is there a way to split them up into 2 files or maybe detect the ide from inside the rc file?

**Comment by AlexPl292** [comment, marked as answer]
Yes, see https://github.com/JetBrains/ideavim#ide-specific-options
